changelog.d/pr-9049

```diff
@@ -0,0 +1,13 @@
+synopsis: Do not always pass --quickjump to haddock #9049
+packages: Cabal
+prs: #9049
+issues: #9060
+description: {
+
+6d8adf13101 caused `cabal` to always pass the `--quickjump` flag to Haddock.
+Not only this wastes memory for a service that user hasn't asked for,
```

Comment:
"Not only does this waste memory", is how I'd say it.

Reply:
Right, thank you!